config/crd/multicluster.x-k8s.io_serviceimports.yaml

```diff
@@ -182,3 +178,61 @@ spec:
                   x-kubernetes-list-map-keys:
                     - cluster
                   x-kubernetes-list-type: map
+                conditions:
```

Comment:
Outch, thanks for spotting those! Could you do a separate PR for your first commit here?